src/internal/serviceenv/service_env.go

```diff
@@ -32,6 +35,10 @@ const clusterIDKey = "cluster-id"
 // create more, if they want to create multiple pachyderm "clusters" served in
 // separate goroutines.
 type ServiceEnv interface {
+	AuthServer() auth_server.APIServer
```

Comment:
Add accessors to ServiceEnv so we can do method calls instead of RPCs between services.